### ISSUE TITLE
development based on PRs even from core devs

### ISSUE THREAD

**Issue by cordoval** [auraphp/Aura.SqlQuery#20]
can we propose so that everybody sends PRs, even core devs, so that we can comment and see better diffs, skimming over commits is not good even though github aggregates the stuff. Please. :baby: thanks!

**Comment by harikt** [comment]
@cordoval not sure this works.

@pmjones is the only person who pushes to core. It is small typo / changes I do talk, and we haven't moved to a long conversations on it.

As github allows it nicely I feel let this be an exception for Paul ;) .

**Comment by cordoval** [comment]
trust me, it does, and no exceptions, only for tags or reverts that could hold, same as in Symfony and elsewhere I have seen.

**Comment by harikt** [comment]
I will wait to hear what @pmjones thinks about the idea. And sometimes you are right when there are more commits on a day hard to follow the changes.

**Comment by harikt** [comment]
I wonder whether we can search the comments inside PR. ( comments made by other developer in between the code in the review process )

**Comment by cordoval** [comment]
of course and they hide when they are addressed, we just did that on the PR you reviewed for me.

**Comment by cordoval** [comment]
review process does not happen on code you write in between, review process happens on the total diff those commits amount, in rare cases you can reference those. but launching a thread in a commit commented is not good imo, if there is a request it can easily get lost.

**Comment by pmjones** [comment]
@cordoval at first i thought the team was too small to really support this, but on thinking about it, i don't see any real problem here. i'll start making my own pmjones forks of the branches and begin working from there. it's a good idea; thanks.

**Comment by harikt** [comment]
:+1:  great Paul.

**Comment by pmjones** [comment]
I have forked all the repos; further commits will come as pull requests.
